api/turing/service/router_deployment_service_test.go

```diff
@@ -248,7 +248,7 @@ func TestDeployEndpoint(t *testing.T) {
 	)
 
 	assert.NoError(t, err)
-	assert.Equal(t, fmt.Sprintf("http://%s-router.models.example.com", routerVersion.Router.Name), endpoint)
+	assert.Equal(t, fmt.Sprintf("http://%s-router.models.example.com/v1/predict", routerVersion.Router.Name), endpoint)
```

Comment:
ah this is due to `https://github.com/caraml-dev/turing/pull/255/files#r987485618`, previously it was done at the json unmarshalling which cause all endpoint to be appended with such, so that logic is moved to the service and constructed here instead.

Although this tests expects `http://%s-router.models.example.com` previously, you will never get it from api calls as during unmarshalling it will become `http://%s-router.models.example.com/v1/predict`